salt/modules/boto_elb.py

```diff
@@ -686,3 +694,223 @@ def get_instance_health(name, region=None, key=None, keyid=None, profile=None, i
     except boto.exception.BotoServerError as error:
         log.debug(error)
         return []
+
+
+def get_listener_policy(name, listener, region=None, key=None, keyid=None, profile=None):
+    '''
+    gets the policy details of a specific listener port
+
+    name
+        name of the elb
+
+    listener
+        the listener port to retrieve details on
```

Comment:
Can you add a CLI Example here? Please see [this comment](https://github.com/saltstack/salt/pull/27548#issuecomment-144766966) for an example of what the syntax should look like.